modules/gapi/src/api/gcomputation.cpp

```diff
@@ -159,16 +159,14 @@ void cv::GComputation::apply(cv::Mat in1, cv::Mat in2, cv::Scalar &out, GCompile
 }
 
 void cv::GComputation::apply(const std::vector<cv::Mat> &ins,
-                             const std::vector<cv::Mat> &outs,
+                             std::vector<cv::Mat> &outs,
```

Comment:
Please be consistent with code alignment here and in the header (personally I prefer alignment from the header)

Reply:
Fixed